### PR TITLE
[docs] Update README.md, clean up build files, add .gitignore

### PR DESCRIPTION
**Pull request by evanw555**
Update `README.md` to fix formatting and to reflect new workflow. Remove unnecessary build files, add entries to `.gitignore`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +27,7 @@ git commit -m 'your comment'
 git push origin [your_branch]
```
Comment (saponniah) [inline README.md]: Once the changes have been merged, do we need a clean-up step to remove the branch from origin?

Comment (evanw555) [reply README.md]: It depends on if the person who merges the PR uses GitHub's option to delete after merge. If not, then yes you are correct. I can add a comment, thanks.

Comment (evanw555) [reply README.md]: Updated

**Comment by saponniah** [comment]
Thanks. Approved and merged the changes.

**Pull request merged**